python/nav/ip.py

```diff
@@ -32,7 +32,7 @@ def __cmp__(self, other):
         try:
             return super(IP, self).__cmp__(other)
```

Comment:
`__cmp__` is also deprecated. Fixed in a later commit maybe?

Reply:
No. This class exists specifically to make IP objects comparable to other types of objects, by overriding functionality from `IPy.IP`. 

Although `__cmp__` is not used by Python 3, the latest version of `IPy.IP` still defers to its `__cmp__` implementation from its `__lt__`-and-friends implementations, so our class overrides __cmp__ as its workaround.